lib/Player.rb

```diff
@@ -0,0 +1,34 @@
+class Player
+    attr_accessor :name, :piece
+    def initialize(name, piece, board)
+        @name = name 
+        @piece = piece
+        @board = board
+    end
+
+    def ask_move
+        loop do
+            puts "Enter move to make "
+            move = gets.chomp
```

Comment:
This code is still UI related. We should separate UI related logic from our classes

Reply:
Still you have a UI related code (`gets.chomp`) in here. We shouldn't be mixing UI logic with our classes